### PR TITLE
teamviewer.xml

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**
v1.5.0

### CODE REVIEW

File: cleaners/teamviewer.xml [hunk 1]
```diff
@@ -12,24 +12,46 @@
...
-    <action command="delete" search="file" path="$APPDATA\TeamViewer\TeamViewer4_Logfile.log"/>
+    <action command="delete" search="file" path="$profile\TeamViewer4_Logfile.log"/>
   </option>
   <option id="mru">
     <label>Most recently used</label>
     <description>Delete the most recently used list</description>
-    <action command="delete" search="file" path="$APPDATA\TeamViewer\Connections.txt"/>
+    <action command="delete" search="walk.files" path="$profile\MRU\"/>
+    <action command="delete" search="glob" path="$profile\MRU\"/>
+    <action command="delete" search="file" path="$profile\Connections.txt"/>
```
Comment (az0) [inline cleaners/teamviewer.xml]: `$$profile$$` here too?

**Comment by az0** [comment]
I am new to using GitHub reviews, so please forgive if I did it wrong, but the other day I noticed the syntax of the multi-variable variable $$profile$$ looks incorrect. This is blocking the merger. If you have any questions about it, let me know.

File: cleaners/teamviewer.xml [hunk 2]
```diff
@@ -12,24 +12,46 @@
...
+    <!--
+    Linux $profile not yet figured out! I will do this maybe later! Tobias.
+    <value search="glob" os="linux">~/.teamviewer</value>
+    -->
+  </var>
   <option id="logs">
     <label>Logs</label>
     <description>Delete the logs</description>
-    <action command="delete" search="file" path="$APPDATA\TeamViewer\TeamViewer4_Logfile.log"/>
+    <action command="delete" search="file" path="$profile\TeamViewer4_Logfile.log"/>
```
[the remark this answers is not in the document]
Comment (Tobias-B-Besemer) [reply cleaners/teamviewer.xml]: Oh, yes !!!

File: cleaners/teamviewer.xml [hunk 3]
```diff
@@ -18,18 +18,40 @@
...
+    @testeddate 2019-03-13
+    @testedby https://github.com/Tobias-B-Besemer
+    @note 
+
 -->
 <cleaner id="teamviewer" os="windows">
   <label>TeamViewer</label>
+  <description>Remote control software</description>
+  <var name="profile">
+    <value search="glob" os="windows">$AppData\TeamViewer</value>
```
Comment (az0) [inline cleaners/teamviewer.xml]: FYI, I am committing to remove the unnecessary search="glob". Not a big deal, but it's not needed

File: cleaners/teamviewer.xml (hunk 1 above)
Comment (Tobias-B-Besemer) [reply cleaners/teamviewer.xml]: Yes!

**Comment by Tobias-B-Besemer** [comment]
> I am new to using GitHub reviews, so please forgive if I did it wrong, but the other day I noticed the syntax of the multi-variable variable $$profile$$ looks incorrect. This is blocking the merger. If you have any questions about it, let me know.

No, no questions! I have ATM just a delay in working of your reviews and give a feedback... :-/

File: cleaners/teamviewer.xml (hunk 3 above)
Comment (Tobias-B-Besemer) [reply cleaners/teamviewer.xml]: What do you mean with "committing to remove" ??? Sorry, maybe my English...
Should I remove it ??? I copied this from somewhere else...

Comment (az0) [reply cleaners/teamviewer.xml]: I made this commit https://github.com/bleachbit/bleachbit/commit/057694a08301b4939d59f7bed5cc996c94a1de52

**Pull request merged**